integrations/docker/images/base/chip-build/Dockerfile

```diff
@@ -183,22 +139,23 @@ RUN set -x \
 RUN set -x \
     && apt-get update \
     # Install build and runtime requirements for IWYU
-    && DEBIAN_FRONTEND=noninteractive apt-get install -fy --fix-missing clang-12 libclang-12-dev \
```

Comment:
I guess that the IWYU is new enough in 24.04, so it's not required to install it from source any more.

Reply:
Updated. We do get version 21 instead of 22, however hopefully good enough.